src/auxiliary/special_elixirs.jl

```diff
@@ -18,12 +18,13 @@ This function assumes that the spatial resolution is set via the keywords
 `initial_refinement_level` (an integer) or `cells_per_dimension` (a tuple of
 integers, one per spatial dimension).
 """
-function convergence_test(mod::Module, elixir::AbstractString, iterations; kwargs...)
+function convergence_test(mod::Module, elixir::AbstractString, iterations, RealT;
```

Comment:
```suggestion
function convergence_test(mod::Module, elixir::AbstractString, iterations, RealT = Float64;
```
Otherwise this would be unnecessarily breaking?